src/status_im2/subs/communities.cljs

```diff
@@ -223,31 +207,62 @@
         (sort-by :position)
         (into []))))
 
+(defn reduce-over-categories
```

Comment:
This needs refactoring, not sure about the naming, but keeping everything into a function was a bit too much on they eye.

I think we should also split the counter, this subscribes to too much data, and it's run every time the count changes/anything in any chat changes.

The counter I think it should be its own component, that subscribes only to the count of that particular chat, that way `full-chats-data` is not pulled in to this subscription.

I will do that in a separate PR if we are happy with it